subprojects/build-operations/src/test/groovy/org/gradle/internal/operations/DefaultBuildOperationRunnerTest.groovy

```diff
@@ -182,8 +182,8 @@ class DefaultBuildOperationRunnerTest extends Specification {
 
         where:
         description        | expectedException             | fail
-        "thrown exception" | new RuntimeException("Error") | { BuildOperationContext context, Exception expectedException -> throw expectedException }
-        "context"          | null                          | { BuildOperationContext context, Exception expectedException -> context.failed(new RuntimeException("Error")) }
+        "thrown exception" | new RuntimeException("Error") | { BuildOperationContext context, Exception e -> throw e }
+        "context"          | null                          | { BuildOperationContext context, Exception e -> context.failed(new RuntimeException("Error")) }
```

Comment:
I'm not sure what exactly changed in Spock and/or whether this is related to StaticCompile, but the compilation fails when using spock2 otherwise: https://ge.gradle.org/s/zy4ecethfbbu4/console-log?anchor=384